fineract-provider/build.gradle

```diff
@@ -225,6 +232,7 @@ bootRun {
     dependencies {
         implementation 'org.mariadb.jdbc:mariadb-java-client'
         implementation 'org.postgresql:postgresql'
+        implementation 'org.pheesdk:PaymentHubSDK:1.0.0'
```

Comment:
I dont think it is right to be here. Would you move to the dependencies.gradle? Also it might be better to have it in a separate module. Most of the ppl would not need payment hub support...

Reply:
The version shouldn't be here that's for sure. In terms of packaging the PaymentHubSDK with Fineract, there are 2 options:
- bundle with Fineract as it is
- extract into a separate module where the payment hub dependency is there - in this case there needs to be build adjustments so that Fineract is published in 2 versions, with and without payment hub

I'm not sure how much extra weight the Payment Hub SDK is, but if it's just DTOs and API classes, I'd say it's acceptable to be bundled with Fineract.